app/controllers/Api/V1/GenresController.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::GenresController < ApplicationController
+  def index
```

Comment:
This request is currently being handled by the FE app, and is not utilized. Hope to implement this down the road.